.github/workflows/gating.yaml

```diff
@@ -28,7 +28,9 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install tox tox-gh-actions
+        pip install tox
+        pip install tox-gh-actions
+        pip install coverage
```

Comment:
github action changes should not be part of this commit. This commit is solely about moving configuration from .coveragerc to pyproject.toml